task_sdk/pyproject.toml

```diff
@@ -27,10 +27,11 @@ dependencies = [
     "httpx>=0.27.0",
     "jinja2>=3.1.4",
     "methodtools>=0.4.7",
-    "msgspec>=0.18.6",
     "psutil>=6.1.0",
     "structlog>=24.4.0",
     "retryhttp>=1.2.0",
+    "orjson>=3.10.11",
+    "msgspec>=0.19.0",
```

Comment:
Do we need this one anymore?

Reply:
Snap https://github.com/apache/airflow/pull/45962#discussion_r1926952505

Reply:
Yep, completely reasonable questions both. We still use it in the execution_time package for a couple of things. We could remove those in a follow up.

Reply:
Reverted back to msgspec in https://github.com/apache/airflow/pull/45962/commits/d43c9e9262c9fc84641ae43144ef9ae48f78f43d